spec/abi.dd

```diff
@@ -697,11 +697,13 @@ $(GNAME TypeTuple):
 
 $(H2 $(LNAME2 function_calling_conventions, Function Calling Conventions))
 
-        $(P The $(D extern (C)) and $(D extern (D)) calling convention matches the C
-        calling convention
-        used by the supported C compiler on the host system.
-        Except that the extern (D) calling convention for Windows x86 is described here.
-        )
+        $(P Functions with $(D extern (C)) linkage matches the C calling
+        convention used by the supported C compiler on the host system.)
```

Comment:
I still think that `used by the supported C compiler on the host system` is to vague